timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -270,6 +290,7 @@ def fit(
         presets: Optional[str] = None,
         hyperparameters: Dict[Union[str, Type], Any] = None,
         hyperparameter_tune_kwargs: Optional[Union[str, Dict]] = None,
+        num_val_windows: int = 1,
```

Comment:
My main issue with this PR is that we appear to be forcing _all_ models to be essentially bagged models with a single parameter. For validation splitter, I think this was OK since this choice was primarily about model selection and validation. Now, it seems to be primarily about training. For example, I could be much more OK doing this for some cheap models and not other more expensive ones?